docs/faq.md

```diff
@@ -119,6 +119,33 @@ Yes, it offers Envoy support in several ways:
 - Connect-gRPC has an [Envoy filter](https://github.com/connectrpc/envoy-demo)
 - Connect-Go supports gRPC health checking via https://github.com/connectrpc/grpchealth-go
 
+### My web request fails but the response looks valid. Why isn't it going through?
+
+Web requests that attempt to load resources from a domain outside the domain of the page
+require you to configure Cross-Origin Resource Sharing (CORS). Not doing so may cause protocol
+errors like "missing trailer", etc. If the response is valid and the request is to a different
+domain, make sure the server has the correct [CORS configuration](./cors.md#configurations-by-protocol).
+
+### Why does my Web client not receive the error code from my server?
+
+Most of the time, it's because the CORS setup of the server is incomplete. It's not enough to allow request origin, 
+methods, and headers—you also have to to expose response headers. See the [CORS documentation](./cors.md) for details.
+
+### How do I set a cookie on the server?
```

Comment:
This answer is specific to Connect-ES, but it's under the section "The Connect protocol".

Can we move it to the section "TypeScript and JavaScript"?